pom.xml

```diff
@@ -19,7 +19,7 @@
         <junit.version>4.13.2</junit.version>
         <karaf.version>4.3.10</karaf.version>
         <log4j.version>2.23.1</log4j.version>
-        <mockito.version>5.11.0</mockito.version>
+        <mockito.version>2.18.0</mockito.version>
```

Comment:
Going back from 5.x to 2.x smells as well.

Reply:
Missed merge in my local branch.  I'll fix it.